controllers/front/ProductController.php

```diff
@@ -326,30 +333,55 @@ public function displayAjaxRefresh()
             'product_customization' => $this->render(
                 'catalog/_partials/product-customization',
                 array(
-                    'customizations' => $product_for_template['customizations'],
+                    'customizations' => $product['customizations'],
                 )
             ),
             'product_details' => $this->render('catalog/_partials/product-details'),
             'product_variants' => $this->render('catalog/_partials/product-variants'),
             'product_discounts' => $this->render('catalog/_partials/product-discounts'),
             'product_add_to_cart' => $this->render('catalog/_partials/product-add-to-cart'),
             'product_url' => $this->context->link->getProductLink(
-                $product_for_template['id_product'],
+                $product['id_product'],
                 null,
                 null,
                 null,
                 $this->context->language->id,
                 null,
-                $product_for_template['id_product_attribute'],
+                $product['id_product_attribute'],
                 false,
                 false,
                 true
             ),
-            'id_product_attribute' => $product_for_template['id_product_attribute'],
+            'product_minimal_quantity' => $minimalProductQuantity,
+            'product_has_combinations' => !empty($this->combinations),
+            'id_product_attribute' => $product['id_product_attribute'],
         )));
     }
 
     /**
+     * Get minimal product quantity or minimal product combination quantity
+     *
+     * @param $product
+     * @return int
+     */
+    protected function getMinimalProductOrDeclinationQuantity($product)
+    {
+        $productAttributeId = $product['id_product_attribute'];
+        $minimalProductQuantity = 1;
+
+        if ($this->combinations) {
+            $minimalCombinationProductQuantity = intval($this->combinations[$productAttributeId]['minimal_quantity']);
+            if ($minimalCombinationProductQuantity) { // Ensure the minimal product combination quantity is not 0;
+                $minimalProductQuantity = $minimalCombinationProductQuantity;
+            }
+        } elseif (array_key_exists('minimal_quantity', $product)) {
```

Comment:
i think we had `else if` everywhere in PrestaShop and it was treated as an standard